### ISSUE TITLE
Implement functionality behind clicking navigation status button

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#36]
## User story
1. As a User
2. I want to be able to see, whether the test app is currently simulating a navigation 
3. So that I know at all times which use case is currently active.

With this user story, it should be implemented that when I click on the navigation status button, I am led back to the currently running navigation use case.

## Acceptance criteria
* It is visible in the app whether the app is currently running a navigation simulation.
* It is visible in the app whether the navigation simulation currently is in the focus of the app (and not running in the background).
* It is possible to see the current status from all screens in the app.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by i315315** [comment]
Duplicate
